docs/sidebars.js

```diff
@@ -1,287 +1,285 @@
-            items: [
-              {
-                type: 'category',
-                label: 'Proto',
-                collapsed: true,
-                items: [
-                  'apis/proto/auth',
-                  'apis/proto/management',
-                  'apis/proto/admin',
-                  'apis/proto/org',
-                  'apis/proto/user',
-                  'apis/proto/app',
-                  'apis/proto/policy',
-                  'apis/proto/auth_n_key',
-                  'apis/proto/change',
-                  'apis/proto/idp',
-                  'apis/proto/member',
-                  'apis/proto/metadata',
-                  'apis/proto/message',
-                  'apis/proto/text',
-                  'apis/proto/object',
-                  'apis/proto/options'
-                ],
-              },
-              {
-                type: 'category',
-                label: 'Assets API',
-                collapsed: true,
-                items: [
-                    'apis/assets/assets',
-                ],
-              },
-            ],
-        },
-        {
-            type: 'category',
-            label: 'OpenID Connect & OAuth',
-            collapsed: false,
-            items: [
-                'apis/openidoauth/endpoints',
-                'apis/openidoauth/scopes',
-                'apis/openidoauth/claims',
-                'apis/openidoauth/authn-methods',
-                'apis/openidoauth/grant-types'
-            ],
-        },
+  quickstarts: [
+    "quickstarts/introduction",
+    {
+      type: "category",
+      label: "Integrate ZITADEL Login in your App",
+      items: [
+        "quickstarts/login/angular",
+        "quickstarts/login/react",
+        "quickstarts/login/flutter",
+        "quickstarts/login/nextjs",
+      ],
+      collapsed: false,
+    },
+    {
+      type: "category",
+      label: "Secure your API",
+      items: ["quickstarts/secure-api/go", "quickstarts/secure-api/dot-net"],
+      collapsed: false,
+    },
+    {
+      type: "category",
+      label: "Call the ZITADEL API",
+      items: [
+        "quickstarts/call-zitadel-api/go",
+        "quickstarts/call-zitadel-api/dot-net",
+      ],
+      collapsed: false,
+    },
+    {
+      type: "category",
+      label: "Identity Aware Proxy",
+      items: ["quickstarts/identity-proxy/oauth2-proxy"],
+      collapsed: false,
+    },
+  ],
+  guides: [
+    "guides/overview",
+    {
+      type: "category",
+      label: "Get to know ZITADEL",
+      collapsed: false,
+      items: [
+        "guides/basics/get-started",
+        "guides/basics/organizations",
+        "guides/basics/projects",
+      ],
+    },
+    // {
+    //   type: "category",
+    //   label: "Architecture Scenarios",
+    //   collapsed: false,
+    //   items: [
+    //     "guides/architecture-scenarios/introduction",
+    //     "guides/architecture-scenarios/b2c",
+    //     "guides/architecture-scenarios/b2b",
+    //   ],
+    // },
```

Comment:
@peintnermax should this not be uncommented?

```suggestion
     {
       type: "category",
       label: "Architecture Scenarios",
       collapsed: false,
       items: [
         "guides/architecture-scenarios/introduction",
         "guides/architecture-scenarios/b2c",
         "guides/architecture-scenarios/b2b",
       ],
     },
```